operator-common/src/main/java/io/strimzi/operator/common/operator/resource/AbstractScalableResourceOperator.java

```diff
@@ -30,9 +30,9 @@
             R extends ScalableResource<T, D>>
         extends AbstractReadyResourceOperator<C, T, L, D, R> {
 
-    public static final String STRIMZI_CLUSTER_OPERATOR_DOMAIN = "operator.strimzi.io";
-    public static final String ANNOTATION_GENERATION = STRIMZI_CLUSTER_OPERATOR_DOMAIN + "/generation";
-    public static final String ANNOTATION_MANUAL_DELETE_POD_AND_PVC = STRIMZI_CLUSTER_OPERATOR_DOMAIN + "/delete-pod-and-pvc";
+    public static final String STRIMZI_OPERATOR_DOMAIN = "operator.strimzi.io";
+    public static final String ANNOTATION_GENERATION = STRIMZI_OPERATOR_DOMAIN + "/generation";
```

Comment:
Is it correct that you changed the annotation also for this one?

Reply:
@scholzj it's just a constant renaming but the value is always "operator.strimzi.io" as the current implementation, isn't it?

Reply:
But the generation annotation is something what has been used before and is largely unrelated to the manual rolling update. And that makes me worried.

Reply:
I cannot understand the concern if the real underlying value is always "operator.strimzi.io/generation"? Am I missing something? Standa just changed the constant name.

Reply:
Its just affecting unrelated functionality which has much wider use.

Reply:
I cannot understand sorry ... it's renaming a constant leaving the same value, so it's about the code not the functionality itself. Btw if you like @stanlyDoge can revert to the old constant name.

Reply:
Is it really just renaming the constant? Or does it change the annotation name from `cluster.operator.strimzi.io` to `operator.strimzi.io` for the generation annotation?

Reply:
Nope, it doesn't change the annotation name even because following a snippet from a running Kafka pod with 0.8.0 rc1 : 

```
"metadata": {
    "annotations": {
      "openshift.io/scc": "restricted",
      "operator.strimzi.io/generation": "0"
    },
```

as you can see it's already using `operator.strimzi.io`

Reply:
Ok then.

Reply:
It is just renaming.